frontend/src/components/edit-profile/edit-profile.tsx

```diff
@@ -102,6 +120,25 @@ export const EditProfile: React.FC<EditProfileProps> = ({ onClose }) => {
     await deleteUserProfile();
   };
 
+  const handleChangeAvatar = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const fileList = e.target.files;
+    if (!fileList) return;
+
+    const file = fileList[0];
+    const validation = ['image/jpeg', 'image/jpg', 'image/png'];
+    const fileSize = Math.round(file.size / 1024);
+    if (!validation.includes(file.type)) {
+      alert('This is not an Image File!');
+    } else if (fileSize > 10240) {
+      alert('File too Big, please select a file less than 10mb');
+    }
```

Comment:
Why don't you stop the execution of the function when the error condition is met? `alert` just shows the message, but below you still create the object URL and set it as avatar